docs/en/Running-Presto-with-Alluxio.md

```diff
@@ -9,52 +9,95 @@ priority: 2
-Alluxio cluster should also be set up in accordance to these guides for either
+* [Setup Java](Java-Setup.html) for Java 8 Update 60 or higher (8u60+), 64-bit.
+* [Deploy Presto](https://prestodb.io/docs/current/installation/deployment.html).
+This doc assumes and is tested with `presto-0.191`.
+* Alluxio cluster should also be set up according to these guides for either
 [Local Mode](Running-Alluxio-Locally.html) or [Cluster Mode](Running-Alluxio-on-a-Cluster.html).
+* Make sure that the Alluxio client jar is available.
+  This Alluxio client jar file can be found at `{{site.ALLUXIO_CLIENT_JAR_PATH}}` in the tarball
+  downloaded from Alluxio [download page](http://www.alluxio.org/download).
+  Alternatively, advanced users can compile this client jar from the source code
+  by following the [instructions](Building-Alluxio-From-Source.html).
+* Make sure that Hive metastore is running to serve metadata information of Hive tables.
+
+## Configuration Presto
+
+### Basic Setup
+
+#### Configure Presto to connect to Hive Metastore
 
-Please [Download Presto](https://repo1.maven.org/maven2/com/facebook/presto/presto-server/)(This doc uses presto-0.191). Also, please complete Hive setup using
-[Hive On Alluxio](Running-Hive-with-Alluxio.html)
+Presto gets the database and table metadata information, as well as
+the file system location of table data from Hive Metastore.
+Edit the Presto configuration `${PRESTO_HOME}/etc/catalog/hive.properties`:
 
-## Configuration
+```properties
+connector.name=hive-hadoop2
+hive.metastore.uri=thrift://localhost:9083
+```
 
-Presto gets the database and table metadata information from Hive Metastore. At the same time,
-the file system location of table data is obtained from the table's metadata entries. So you need to configure
-[Presto on HDFS](https://prestodb.io/docs/current/installation/deployment.html). In order to access HDFS,
-you need to add the Hadoop conf files (core-site.xml,hdfs-site.xml), and use `hive.config.resources` in
-file `/<PATH_TO_PRESTO>/etc/catalog/hive.properties` to point to the file's location for every Presto worker.
+#### Distribute the Alluxio client jar to all Presto servers
 
-### Configure `core-site.xml`
+Put Alluxio client jar `{{site.ALLUXIO_CLIENT_JAR_PATH}}` into directory
+`${PRESTO_HOME}/plugin/hive-hadoop2/`
+(this directory may differ across versions) on all Presto servers. Restart the process of
+coordinator and worker.
 
-You need to add the following configuration items to the `core-site.xml` configured in `hive.properties`:
+### Advanced Setup
+
+#### Configure additional Alluxio properties
+
+To configure additional Alluxio properties, one can add them to the Hadoop conf files 
+(`core-site.xml`, `hdfs-site.xml`), and use 
+Presto property `hive.config.resources` in
+file `${PRESTO_HOME}/etc/catalog/hive.properties` to point to the file's location for every Presto
+worker.
+
+```
+hive.config.resources=/<PATH_TO_CONF>/core-site.xml, /<PATH_TO_CONF>/hdfs-site.xml
```

Comment:
just double-checking, is there supposed to be a space after the comma?

Reply:
space removed